crates/bevy_reflect/bevy_reflect_derive/Cargo.toml

```diff
@@ -23,3 +23,7 @@ syn = { version = "2.0", features = ["full"] }
 proc-macro2 = "1.0"
 quote = "1.0"
 uuid = { version = "1.1", features = ["v4"] }
+bit-set = "0.5.2"
```

Comment:
This feels out of place, could it be a remnant of a previous merge?

Reply:
fixed, thx